API/Controllers/CategoryController.cs

```diff
@@ -67,13 +73,33 @@ public async Task<ActionResult<IEnumerable<CategoryDto>>> GetUserCategories()
             return Ok(categories);
         }
 
+        [HttpGet("{categoryId}")]
+        public async Task<ActionResult<CategoryDto>> GetUserCategory(int categoryId)
+        {
+            var category =  await _categoryRepository.GetCategoryAsync(categoryId);
+            return Ok(category);
+        }
+
+        [HttpGet("pagination")]
```

Comment:
Bez sensu w sciezce dawac pagination, co to znaczy. Po prostu get to jest. Jak masz jakis endpoint od tego samego ale bez paginacji to pytanie brzmi po co, można je złączyć w jeden